### PR TITLE
Add extra labels in jinja template

### PR DESCRIPTION
**Pull request by miguel-chacon**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions
* Remember to set **idealista:develop** as base branch;

### Description of the Change
Add all non mandatory fields in `extra_labels` to use them in jinja2 template. If you want to ignore any of these extra fields add them to config as coma-separated list.

### Benefits
A more flexible template

### Possible Drawbacks
None

### Applicable Issues
#112

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,13 @@ This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a ch
 ## Changed
 - *[#110](https://github.com/idealista/prom2teams/pull/110) Using Alpine Linux instead of Debian Slim in Docker Image* @dortegau
 
+## [2.3.1](https://github.com/idealista/prom2teams/tree/2.3.1)
+[Full Changelog](https://github.com/idealista/prom2teams/compare/2.3.0...2.3.1)
+## Changed
+- *[#112](https://github.com/idealista/prom2teams/issues/112) Allow custom labels in jinja template*
+- *Update flask-restplus version to v0.12.1*
+- *Update marshmallow version to v3.0.0rc6*
```
Comment (dortegau) [inline CHANGELOG.md]: I would prefer only stable versions :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,13 @@ This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a ch
 ## Changed
 - *[#110](https://github.com/idealista/prom2teams/pull/110) Using Alpine Linux instead of Debian Slim in Docker Image* @dortegau
 
+## [2.3.1](https://github.com/idealista/prom2teams/tree/2.3.1)
+[Full Changelog](https://github.com/idealista/prom2teams/compare/2.3.0...2.3.1)
+## Changed
+- *[#112](https://github.com/idealista/prom2teams/issues/112) Allow custom labels in jinja template*
```
Comment (dortegau) [inline CHANGELOG.md]: Could you add your name to that changes?

Comment (miguel-chacon) [reply CHANGELOG.md]: Done

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,13 @@ This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a ch
 ## Changed
 - *[#110](https://github.com/idealista/prom2teams/pull/110) Using Alpine Linux instead of Debian Slim in Docker Image* @dortegau
```
[the remark this answers is not in the document]
Comment (miguel-chacon) [reply CHANGELOG.md]: Those changes are already in this release so I'll update changelog

**Pull request merged**